# fix location of libXfont2.so and libfontenc.so
# don't derive location of libpixman-1.so from path to libX11.so
# author: Kenneth Hoste (HPC-UGent)
#
# updated to v3.0.1 by maxim-masterov (SURF)

diff -Nru turbovnc-3.0.1.orig/unix/Xvnc/CMakeLists.txt turbovnc-3.0.1/unix/Xvnc/CMakeLists.txt
--- turbovnc-3.0.1.orig/unix/Xvnc/CMakeLists.txt	2022-09-22 11:31:39.142872000 +0200
+++ turbovnc-3.0.1/unix/Xvnc/CMakeLists.txt	2022-09-22 11:33:07.984771000 +0200
@@ -96,9 +96,8 @@
 if(TVNC_SYSTEMX11)
 	include_directories(${X11_X11_INCLUDE_PATH} ${X11_Xau_INCLUDE_PATH}
 		${X11_Xdmcp_INCLUDE_PATH} ${X11_Xkbfile_INCLUDE_PATH})
-	string(REGEX REPLACE "X11" "Xfont2" X11_Xfont2_LIB ${X11_X11_LIB})
-	string(REGEX REPLACE "X11" "fontenc" X11_Fontenc_LIB ${X11_X11_LIB})
-	string(REGEX REPLACE "X11" "pixman-1" X11_Pixman_LIB ${X11_X11_LIB})
+	string(REGEX REPLACE "libX11" "libXfont2" X11_Xfont2_LIB ${X11_X11_LIB})
+	string(REGEX REPLACE "libX11" "libfontenc" X11_Fontenc_LIB ${X11_X11_LIB})
 else()
 	include_directories(${CMAKE_CURRENT_BINARY_DIR}/X_include)
 	set(X11_Xau_LIB Xau)
